### DISCUSSION TITLE
Who made the nim prompt theme?

### DISCUSSION THREAD

**Discussion by TehDomic** [fish-shell/fish-shell#10341] [Q&A] [answered]
I'd like to use it for PowerShell to try emulate my fish setup.

**Comment by faho** [comment, marked as answer]
[The comment at the top](https://github.com/fish-shell/fish-shell/blob/ff6fd699fe90b584a7c3bb162f454d5d39ee3512/share/tools/web_config/sample_prompts/nim.fish#L1-L2) says it came from https://github.com/nim65s/dotfiles/.
